website/src/pages/product/automation.tsx

```diff
@@ -0,0 +1,231 @@
+                        />
+                    </div>
+                    <div className="col-md-6">
+                        <Blockquote
+                            quote='[At Facebook], analyzers participate as bots in code review, making automatic comments when an engineer submits a code modification. … [This automation] saw a 70% fix rate, where a more traditional "offline" or "batch" deployment (where bug lists are presented to engineers, outside their workflow) saw a 0% fix rate.'
+                            by={
+                                <a
+                                    href="https://cacm.acm.org/magazines/2019/8/238344-scaling-static-analyses-at-facebook/fulltext"
+                                    target="_blank"
+                                >
+                                    Facebook paper in Communications of the ACM
+                                </a>
+                            }
+                        />
+                    </div>
+                </div>
+                <h2 id="automation-helps-teams-move-more-quickly" className="text-center display-4 mb-3 mt-4">
+                    How automation helps teams move quickly and safely
+                </h2>
+                <div className="row mt-4">
+                    <div className="col-lg-4 mb-4">
+                        <IconItem className="position-relative" icon={DeleteSweepIcon} color="brand-cyan">
+                            <h4>Remove deprecated code and legacy systems</h4>
+                            <p>
+                                You need a way to improve and change APIs used across all of your organization's code,
+                                to spend less time and effort in the migration period between the old and new API or
+                                service.
+                            </p>
+                        </IconItem>
+                    </div>
+                    <div className="col-lg-4 mb-4">
+                        <IconItem className="position-relative" icon={SecurityIcon} color="brand-purple">
+                            <h4>Triage and follow-through on critical security issues</h4>
+                            <p>
+                                You need to be able to identify everywhere that a vulnerable package or API is used, and
+                                open issues or pull requests on all affected projects. Then you can monitor the progress
+                                of fixing, merging, and deploying.
+                            </p>
+                        </IconItem>
+                    </div>
+                    <div className="col-lg-4 mb-4">
+                        <IconItem className="position-relative" icon={NotificationsActiveIcon} color="green">
+                            <h4>Dependency updates</h4>
+                            <p>
+                                Keep your library dependencies **and how you use those libraries** up-to-date and
```

Comment:
```suggestion
                                Keep your library dependencies <i>and how you use those libraries</i> up-to-date and
```

Reply:
(Not sure if this was intentional, but at first glance it looked like you intended to have bold or italics, and the markdown just wasn't converting)